### PR TITLE
Set podSubnet to match calico config

### PR DESCRIPTION
**Pull request by detiber**
Resolves: #147

### CODE REVIEW

**Comment by detiber** [comment]
@chuckha @alexbrand This updates the quickstart to configure the PodSubnet when using calico and resolves the reported networking issue.

File: scripts/setup-k8s-master.sh.in [hunk 1]
```diff
@@ -53,6 +54,13 @@ apiServerCertSANs:
 - ${LB_DNS}
 EOF
 
+if [[ "{{NetworkingProvider}}" == "calico" ]]; then
+cat >>/tmp/kubeadm.yaml <<EOF
+networking:
+  podSubnet: 192.168.0.0/16
```
Comment (alexbrand) [inline scripts/setup-k8s-master.sh.in]: Should this _always_ be set, regardless of CNI plugin?

Comment (detiber) [reply scripts/setup-k8s-master.sh.in]: @alexbrand No, for example it should not be set for weave and flannel requires a different value to be set: https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network

I still need to verify if we need to set the sysctl for net.bridge.bridge-nf-call-iptables when configuring weave, as that page suggests.

**Comment by chuckha** [comment]
yeah this looks good to me. i was going to ask that @alexbrand, so thanks for saving me time :)

**Pull request merged**